tests/keras/backend/backend_test.py

```diff
@@ -4,12 +4,28 @@
 import scipy.sparse as sparse
 
 from keras import backend as K
-from keras.backend import theano_backend as KTH, floatx, set_floatx, variable
-from keras.backend import tensorflow_backend as KTF
+from keras.backend import floatx, set_floatx, variable
 from keras.utils.conv_utils import convert_kernel
-from keras.backend import cntk_backend as KC
 
-BACKENDS = [KTH, KTF, KC]
+BACKENDS = []  # Holds a list of all available back-ends
+
+try:
+    from keras.backend import cntk_backend as KC
+    BACKENDS.append(KC)
+except:
+    print("Could not import the CNTK back-end")
```

Comment:
This should be a warning. Also, please use single quotes. And "backend" in one word.